websocket_managed_conn_test.go

```diff
@@ -16,6 +16,35 @@ const (
 	testToken   = "TEST_TOKEN"
 )
 
+func TestRTMBeforeEvents(t *testing.T) {
```

Comment:
test is unrelated. but it test that disconnect will work even when manage connection is run after.